app/views/readings/index.html.erb

```diff
@@ -0,0 +1,13 @@
+
+<h1>All Reading in #<%= params[:sensor_id] %>, <%= Sensor.find(params[:sensor_id]).sensorType%></h1>
```

Comment:
You shouldn't access data directly from the view, always pass it from a controller method to the view: `Sensor.find(params[:sensor_id]).sensorType`